netlib/http2/__init__.py

```diff
@@ -0,0 +1,182 @@
+from __future__ import (absolute_import, print_function, division)
+import itertools
+import logging
+
+from .frame import *
+from .. import utils
+
+log = logging.getLogger(__name__)
```

Comment:
@cortesi: Probably a dumb question - how does this play with mitmproxy/mitmdump?